libraries/ESP8266WiFi/examples/BearSSL_MaxFragmentLength/BearSSL_MaxFragmentLength.ino

```diff
@@ -17,17 +17,22 @@ const char *pass = STAPSK;
 void fetch(BearSSL::WiFiClientSecure *client) {
   client->write("GET / HTTP/1.0\r\nHost: tls.mbed.org\r\nUser-Agent: ESP8266\r\n\r\n");
   client->flush();
-  uint32_t to = millis() + 5000;
+  uint32_t startMillis = millis();
```

Comment:
Can we _please_ get this millis check ported to polledTimeout? Something like:
```cpp
using oneShot = esp8266::polledTimeout::oneShot;
oneShot timeout(5000);
do {
...
} while(!timeout);
```

Reply:
I changed the loop into using polledTimeout